common/source/docs/common-cabling-guide.rst

```diff
@@ -0,0 +1,275 @@
+.. _common-cabling-guide:
+
+========================
+Cable Design Guidelines
+========================
+
+This guide provides detailed recommendations for cabling with a focus on signal integrity, electromagnetic compatibility, wire color conventions, connector standards, and best practices for specific signals like I2C and CAN. These practices help ensure reliable operation, simplify debugging, and improve documentation clarity across unmanned platforms of all sizes.
+
+Cable Colour Coding
+-------------------
+
+A consistent cable colour-coding system is usefull for correct assembly and maintenance. It also improves clarity in documentation and field repairs.
+
+.. list-table:: Recommended Cable Colour Coding
+   :header-rows: 1
+   :widths: 15 30 55
```

Comment:
this is a build error...only two columns in the table

Reply:
Thanks, fixed in code.